lib/jsonld_serializers.js

```diff
@@ -272,6 +276,28 @@ ResourceSerializer.formatElectronicResourceBlankNode = function (link, rdfsType)
   }
 }
 
+ResourceSerializer.formatItemFilterAggregations = function (aggregations) {
+  return Object.keys(aggregations).map((aggregation) => {
+    const field = aggregation.slice(5)
+    const values = aggregations[aggregation]._nested.buckets.map((bucket) => {
+      const isFormatLiteral = field === 'format'
+      const [value, label] = bucket.key.split('||')
+      return {
+        value: isFormatLiteral ? bucket.key : value,
+        count: bucket.doc_count,
+        label: isFormatLiteral ? bucket.key : label
+      }
```

Comment:
This was a little tricky for me to follow bc the relationship between bucket.key and value is unclear. Consider establishing `value` and `label` defaults and branching to handle packed values when detected
```
let value = bucket.key
let label = value
// If it's a packed value, parse out the value and label:
if (value.split('||').length === 2) {
  ; [value, label] = value.split('||')
}
```

Reply:
👍